### PR TITLE
YACHT-1002: Moved SLI metadata creation from bbq to metadata storage project

### PR DESCRIPTION
**Pull request by przemyslaw-jasinski**

### CODE REVIEW

File: terraform/sli/SLI_history_dataset.tf [hunk 1]
```diff
@@ -26,6 +26,7 @@ resource "google_bigquery_table" "SLI_backup_creation_latency" {
 
   time_partitioning {
     type = "DAY"
+    expiration_ms = 31536000000 // one year
```
Comment (marek-tabor) [inline terraform/sli/SLI_history_dataset.tf]: you could use locals

File: terraform/sli/vars.tf [hunk 2]
```diff
@@ -6,6 +6,10 @@ variable "bbq_restoration_project" {
   description = "it is needed to filter out restored backups from SLI"
 }
 
+variable "bbq_metadata_project" {
+  description = "storage project for bbq metadatas"
```
Comment (marek-tabor) [inline terraform/sli/vars.tf]: metadata without "s"

Comment (przemyslaw-jasinski) [reply terraform/sli/vars.tf]: Done

File: terraform/sli/SLI_history_dataset.tf (hunk 1 above)
Comment (przemyslaw-jasinski) [reply terraform/sli/SLI_history_dataset.tf]: Done

File: SETUP.md [hunk 3]
```diff
@@ -83,13 +83,8 @@ To perform backup, BBQ needs rights to read BigQuery data from the project which
 * Grant this permission for the whole folder or organisation. It will be inherited by all of the projects underneath.
 
 ### Cloud Datastore export
-  BBQ may periodically export data from Datastore to Big Query. It's much easier to query the data in Big Query rather than Datastore. To enable export:
-
-* Execute command below which assigns the **Cloud Datastore Import Export Admin** IAM role to BBQ default service account:
```
Comment (radkomateusz) [inline SETUP.md]: What about removed command? I think that BBQ default account should still have Cloud Datastore Import Export Admin role on BBQ project to be able to export data from datastore, and also have additional accesses to be able export that data outside of bbq project. 

Why we want to remove that from documentation?

File: src/commons/config/configuration.py [hunk 4]
```diff
@@ -38,6 +38,10 @@ def backup_settings_custom_project_list(self):
     def backup_project_id(self):
         return self.__project_config['project_settings'].get('backup_project_id')
 
+    @property
+    def metadata_storage_project_id(self):
+        return self.__project_config['project_settings'].get('metadata_storage_project_id')
```
Comment (radkomateusz) [inline src/commons/config/configuration.py]: we have unit tests for configuration, please add test for new configuration entry

File: terraform/bbq/project_iams.tf [hunk 5]
```diff
@@ -36,3 +36,21 @@ resource "google_project_iam_member" "project_bigqueryjobuser_iam" {
   role    = "roles/bigquery.jobUser"
   member  = "serviceAccount:${var.bbq_project}@appspot.gserviceaccount.com"
 }
+
+resource "google_project_iam_member" "project_bigquerydataeditor_iam" {
+  project = "${var.bbq_metadata_project}"
+  role    = "roles/bigquery.dataEditor"
+  member  = "serviceAccount:${var.bbq_project}@appspot.gserviceaccount.com"
+}
```
Comment (radkomateusz) [inline terraform/bbq/project_iams.tf]: does this roles allow bbq service account to perform export?
in other words, does it the reason why you removed command assigining Datastore Export... role from documentation?

File: SETUP.md (hunk 3 above)
Comment (przemyslaw-jasinski) [reply SETUP.md]: Discusses offline :)

File: src/commons/config/configuration.py (hunk 4 above)
Comment (przemyslaw-jasinski) [reply src/commons/config/configuration.py]: Done

File: terraform/bbq/project_iams.tf (hunk 5 above)
Comment (przemyslaw-jasinski) [reply terraform/bbq/project_iams.tf]: Discusses offline :)

**Pull request merged**